basics.tex

```diff
@@ -1316,8 +1316,29 @@ \section{Cartesian product types}
   {{p \ct q}}
   &= {\pairpath \big({\projpath{1} (p)} \ct {\projpath{1} (q)},\,{\projpath{2} (p)} \ct {\projpath{2} (q)}\big)}.
 \end{align*}
-The same is true for the rest of the higher groupoid structure considered in \cref{sec:equality}.
+Or, written differently:
+\begin{alignat*}{2}
+  \projpath{i}(\refl{(z : A \times B)}) &= \refl{\proj{i} z} &\qquad (i=1,2)\\
+  \pairpath(\opp p, \opp q) &= \opp{\pairpath(p,q)}\\
+  \pairpath(p\ct q, p'\ct q') &= \pairpath(p,p') \ct \pairpath(q,q').
+\end{alignat*}
 All of these equations can be derived by using path induction on the given paths and then returning reflexivity.
+The same is true for the rest of the higher groupoid structure considered in \cref{sec:equality}, although it begins to get tedious to insert enough other coherence paths to yield an equation that will typecheck.
+For instance, if we denote the path in \cref{thm:omg}\ref{item:omg4} by $\ctassoc(p,q,r)$ and the last path displayed above by $\pairct(p,q,p',q')$, then for any $u,v,z,w:A\times B$ and $p,q,r,p',q',r'$ of appropriate types we have
+\begin{equation*}
+  \begin{array}{l}
```

Comment:
I thought it would look better to align the `\ct` symbols on each side of the equation separately, so the reader can easily separate the six ingredient paths.  But I'm not attached to it; if you can cook up something using an ams environment that looks better, be my guest!